### DISCUSSION TITLE
How to handle decimal values for nCount_RNA.

### DISCUSSION THREAD

**Discussion by ksaunders73** [satijalab/seurat#4220] [Q&A] [answered]
My dataset is very strange. When I create the Seurat object and load the metadata for it, all of the values in the nCount_RNA are decimal values instead of integers. How should I interpret this? Is there an issue with the data itself or something I can do to work around this? I ask because later on in my analysis, the functions can't seem to find the nCount_RNA object, and I believe the decimal values are the reason why. 

![image](https://user-images.githubusercontent.com/62159599/111054284-0a89d080-8431-11eb-96f5-d171d8f2978b.png)

Hopefully this isn't too vague, and thanks for reading!

**Comment by andrewwbutler** [comment, marked as answer]
Hi,

My guess here is that the input matrix you used when creating the Seurat object contained non-integer data.
